adcc/guess/__init__.py

```diff
@@ -28,6 +28,23 @@
            "guesses_spin_flip", "guess_symmetries"]
 
 
+def guess_kwargs_kind(kind):
+    """
+    Return the kwargs required to be passed to `guesses_from_diagonal` to
+    computed states of the passed excitation `kind`.
+    """
+    if kind == "singlet":
+        return dict(spin_block_symmetrisation="symmetric", spin_change=0)
+    elif kind == "triplet":
+        return dict(spin_block_symmetrisation="antisymmetric", spin_change=0)
+    elif kind == "spin_flip":
+        return dict(spin_block_symmetrisation="none", spin_change=-1)
+    elif kind == "any":
+        return dict(spin_block_symmetrisation="none", spin_change=0)
```

Comment:
More pythonic to use a dict for the mapping instead?

Reply:
indeed.